radlab-ui/webapp/src/components/forms/StepCreator.tsx

```diff
@@ -46,9 +46,7 @@ const StepCreator: React.FC<StepCreator> = ({ variableList, idx }) => {
     }
 
     // Required checks
-    if (!variable.required) return null
-    if (value || value === "0") return null
-    if (!variable.default) return null
+    if (!variable.required || value || value === "0") return null
```

Comment:
Do we need 3 checks here if we have new logic (in the `terraform.ts`) that properly set this? Can we just do `if (!variable.required) return null`?

Reply:
it's required we if we keep only `(!variable.required)`  then while typing error message not disappearing